sklearn_bench/dbscan.py

```diff
@@ -40,17 +39,17 @@ def main():
     labels = dbscan.labels_
 
     params.n_clusters = len(set(labels)) - (1 if -1 in labels else 0)
-    acc = davies_bouldin_score(X, labels)
+    acc = bench.davies_bouldin_score(X, labels)
 
     bench.print_output(library='sklearn', algorithm='dbscan', stages=['training'],
                        params=params, functions=['DBSCAN'], times=[time],
-                       accuracies=[acc], accuracy_type='davies_bouldin_score',
+                       metrics=[acc], metric_type='davies_bouldin_score',
                        data=[X], alg_instance=dbscan)
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description='scikit-learn DBSCAN benchmark')
-    parser.add_argument('-e', '--eps', '--epsilon', type=float, default=10.,
+    parser.add_argument('-e', '--eps', '--epsilon', type=float, default=0.5,
```

Comment:
why did you change it? this will not affect the measurements of the current configs?

Reply:
It is default value in sklearn

Reply:
this will not affect the measurements of the current configs?